### ISSUE TITLE
Create databases

### ISSUE THREAD

**Issue by muer123** [muer123/2020-GPW-L5K1-B#8]
I think you should be more active so that you have more time to revise.

**Comment by muer123** [comment]
Create all the data with one person's account. use Allen?

**Comment by Alllenisxuhaixiang** [comment]
OK. Ian, Nelson and I use phpMyAdmin tool to create some table data in my database according to the ERD. And we will improve and modify it in the last few days.

**Comment by muer123** [comment]
Yes, it should add more specific data. for example, display the different pagement information.
